lib/index.js

```diff
@@ -1,343 +1,381 @@
+                                    }
 
-                                return ';';
+                                    break;
                             }
 
-                            break;
+                            return '';
+                        });
 
-                        case 'drop':
+                        Fs.readdirSync(`${__dirname}/hbs/partials/`)
+                            .forEach((fileName) => {
 
-                            break;
-                    }
+                                Handlebars.registerPartial(
+                                    fileName.split('.hbs')[0],
+                                    String(Fs.readFileSync(`${__dirname}/hbs/partials/${fileName}`))
+                                );
+                            });
 
-                    return '';
-                });
+                        const compiler = Handlebars.compile(String(Fs.readFileSync(__dirname + '/hbs/migrationFileTemplate.hbs')));
+                        const createdTables = tableDeltas.tableDelta.create.concat(tableDeltas.joinTableDelta.create);
 
-                Fs.readdirSync(`${__dirname}/hbs/partials/`)
-                    .forEach((fileName) => {
+                        // Need this message for both cases where no migration is needed
+                        // (because unsupported columns get skipped)
+                        // and on a successful migration file generation where other
+                        // columns are changed or tables added, etc.
+                        let skippedColsMsg = '';
+                        if (!!internals.skippedCols.length) {
+                            const dedupedSkippedCols = internals.skippedCols.filter((skippedInfo, i) => {
 
-                        Handlebars.registerPartial(
-                            fileName.split('.hbs')[0],
-                            String(Fs.readFileSync(`${__dirname}/hbs/partials/${fileName}`))
-                        );
-                    });
+                                return internals.skippedCols.indexOf(skippedInfo) === i;
+                            });
+                            skippedColsMsg = `Skipped unsupported columns:${Os.EOL}${dedupedSkippedCols.join(Os.EOL)}`;
+                        }
 
-                const compiler = Handlebars.compile(String(Fs.readFileSync(__dirname + '/hbs/migrationFileTemplate.hbs')));
-                const createdTables = tableDeltas.tableDelta.create.concat(tableDeltas.joinTableDelta.create);
+                        if (createdTables.length === 0 &&
+                            columnDeltas.changes.length === 0) {
 
-                if (createdTables.length === 0 &&
-                    columnDeltas.changes.length === 0) {
+                            const noMigrationMsg = !!skippedColsMsg ? `${skippedColsMsg}${Os.EOL}No migration needed` : 'No migration needed';
+                            return cb(null, noMigrationMsg);
```

Comment:
Ideally return values of this function would be structured rather than strings.  For example,
```js
{
    code: NO_MIGRATION,
    skippedColumns: ['latitude', 'longitude']
}
```